protocol.md

```diff
@@ -1323,6 +1324,45 @@ interface LogMessageParams {
 
 Where type is defined as above.
 
+#### <a name="window_progress"></a>Progress Notification
+
+The progress notification is sent from the server to the client to ask the client to indicate progress.
+
+_Notification_:
+* method: 'window/progress'
+* params: `ProgressParams` defined as follows:
+
+```typescript
+interface ProgressParams {
+  /**
+   * A unique identifier to associate multiple progress notifications with the same progress.
+   */
+  id: string;
+
+  /**
+   * The title of the progress.
+   * This should be the same for all ProgressParams with the same id.
+   */
+  title: string;
+
+  /**
+   * Optional progress message to display.
```

Comment:
This isn't the title, this is the message. The title is required, the message is optional. See https://sourcegraph.com/github.com/Microsoft/vscode@1.12.1/-/blob/src/vs/workbench/api/node/extHost.api.impl.ts#L356